packages/host/app/services/card-service.ts

```diff
@@ -257,23 +255,19 @@ export default class CardService extends Service {
     patchData: PatchData,
   ): Promise<CardDef | undefined> {
     let api = await this.getAPI();
-    let initialDoc = await this.serializeCard(card);
-    let updatedCard = await api.updateFromSerialized<typeof CardDef>(card, doc);
     let linkedCards = await this.loadPatchedCards(patchData, new URL(card.id));
     for (let [field, value] of Object.entries(linkedCards)) {
-      // TODO this triggers a save which is not ideal. perhaps instead we could
+      if (field.includes('.') || Array.isArray(value)) {
+        // TODO [wip] linksToMany and nested linksTo
+        throw new Error('Not implemented.');
+      }
```

Comment:
probably we should file a ticket for this scenario. or I suppose if you run into it in while you apply patches with linksToMany from the ai bot work you can handle it here too